<?xml version="1.0" encoding="UTF-8"?>
<!-- Generated by org.testng.reporters.JUnitReportReporter -->
<<<<<<< HEAD
<testsuite skipped="0" hostname="LH7U05CG6422QSW" name="runners.Mapping_Configuration_Runner" tests="1" failures="0" timestamp="20 Sep 2017 11:50:55 GMT" time="0.010" errors="0">
  <testcase name="feature" time="0.010" classname="runners.Mapping_Configuration_Runner"/>
=======
<testsuite skipped="0" hostname="LH7U05CG6272CBT" name="runners.Mapping_Configuration_Runner" tests="1" failures="0" timestamp="03 Oct 2017 07:56:54 GMT" time="73.213" errors="1">
  <testcase name="feature" time="73.213" classname="runners.Mapping_Configuration_Runner">
    <error type="cucumber.runtime.CucumberException" message="java.lang.AssertionError: null">
      <![CDATA[cucumber.runtime.CucumberException: java.lang.AssertionError: null
	at cucumber.api.testng.TestNGCucumberRunner.runCucumber(TestNGCucumberRunner.java:69)
	at cucumber.api.testng.AbstractTestNGCucumberTests.feature(AbstractTestNGCucumberTests.java:21)
	at sun.reflect.NativeMethodAccessorImpl.invoke0(Native Method)
	at sun.reflect.NativeMethodAccessorImpl.invoke(NativeMethodAccessorImpl.java:62)
	at sun.reflect.DelegatingMethodAccessorImpl.invoke(DelegatingMethodAccessorImpl.java:43)
	at java.lang.reflect.Method.invoke(Method.java:498)
	at org.testng.internal.MethodInvocationHelper.invokeMethod(MethodInvocationHelper.java:108)
	at org.testng.internal.Invoker.invokeMethod(Invoker.java:661)
	at org.testng.internal.Invoker.invokeTestMethod(Invoker.java:869)
	at org.testng.internal.Invoker.invokeTestMethods(Invoker.java:1193)
	at org.testng.internal.TestMethodWorker.invokeTestMethods(TestMethodWorker.java:126)
	at org.testng.internal.TestMethodWorker.run(TestMethodWorker.java:109)
	at org.testng.TestRunner.privateRun(TestRunner.java:744)
	at org.testng.TestRunner.run(TestRunner.java:602)
	at org.testng.SuiteRunner.runTest(SuiteRunner.java:380)
	at org.testng.SuiteRunner.runSequentially(SuiteRunner.java:375)
	at org.testng.SuiteRunner.privateRun(SuiteRunner.java:340)
	at org.testng.SuiteRunner.run(SuiteRunner.java:289)
	at org.testng.SuiteRunnerWorker.runSuite(SuiteRunnerWorker.java:52)
	at org.testng.SuiteRunnerWorker.run(SuiteRunnerWorker.java:86)
	at org.testng.TestNG.runSuitesSequentially(TestNG.java:1301)
	at org.testng.TestNG.runSuitesLocally(TestNG.java:1226)
	at org.testng.TestNG.runSuites(TestNG.java:1144)
	at org.testng.TestNG.run(TestNG.java:1115)
	at org.testng.remote.AbstractRemoteTestNG.run(AbstractRemoteTestNG.java:132)
	at org.testng.remote.RemoteTestNG.initAndRun(RemoteTestNG.java:230)
	at org.testng.remote.RemoteTestNG.main(RemoteTestNG.java:76)
Caused by: java.lang.AssertionError: null
	at org.testng.Assert.fail(Assert.java:93)
	at org.testng.Assert.fail(Assert.java:100)
	at stepDefinitions.Mapping_Configuration.ige_Mapping_Configuration_Page_Click_on_Edit_link(Mapping_Configuration.java:144)
	at âœ½.When IGE Mapping Configuration Page: Click on Edit link(./Features/US747422 - Connect UI mapping search layout to model - MAPS Converter UI.feature:27)
]]>
    </error>
  </testcase> <!-- feature -->
>>>>>>> lokesh
</testsuite> <!-- runners.Mapping_Configuration_Runner -->
